clang/lib/Sema/SemaSYCL.cpp

```diff
@@ -2682,19 +2696,47 @@ void SYCLIntegrationHeader::emit(raw_ostream &O) {
   }
   O << "};\n\n";
 
+  O << "// array representing kernel parameters used in all kernels defined in "
+       "the\n";
+  O << "// corresponding source\n";
+  O << "static constexpr\n";
+  O << "const bool param_omit_table[] = {\n";
+  O << "  //OMIT_TABLE_BEGIN\n";
```

Comment:
Ah, it was the trailing spaces I was talking about.  Space between the // and OMIT both still probably need to be there:
```suggestion
  O << "  // OMIT_TABLE_BEGIN\n";
```